Readme.md

```diff
@@ -14,7 +14,9 @@ Do you want to quickly try out this project and see it in action? Check it out,
 
 ## Pre-requisites
 
-You must have [Node.js](https://nodejs.org/) installed for the build to succeed. On Unix-like systems we suggest installing it as user, preferably via the [Node Version Manager](https://github.com/nvm-sh/nvm).
+- You must have [Node.js](https://nodejs.org/) installed for the build to succeed.
+  - On Unix-like systems we suggest installing it as user, preferably via the [Node Version Manager](https://github.com/nvm-sh/nvm).
+- You must have [PNPM](https://pnpm.io/) installed with `npm install --global pnpm`.
```

Comment:
Yours has more accurate information, but I find it unnecessarily wordy. A user looks at this section to see what must be installed. No need for the marketing material to get them on board with each tool.

Reply:
I will revert it for now, to expedite this issue's completion, but I still urge you to consider rewriting the section in a bullet-list format because I don't think prose is very user-friendly for the purposes of the text.

Reply:
Sorry for not answering. I actually agree with you! It's just important that right now NE only supports **npm@6**.

Reply:
Great, thanks! But don't you mean `pnpm@latest-6` instead? That's the tag I see, when I look at the versions tab [here](https://www.npmjs.com/package/pnpm).

Now that I think about it, could you explain what is meant by "If you're using `Node.js` 16.9 or later, `pnpm` will be used automatically." [here](https://github.com/Lombiq/NodeJs-Extensions/commit/e6c6106f2fe647ded52a973c7e93b1c42c8b9e0c#diff-1550ec65ac92f65817fc28928dfef526912b5f52356ff43651369bae92f56031R21)? On the [pnpm installation guide](https://pnpm.io/installation#using-corepack) it says you need v16.13, not v16.9. And even then you have to actually enable it with `corepack enable; corepack prepare pnpm@latest-6 --activate`. Or am I missing something?

Reply:
> But don't you mean `pnpm@latest-6` instead?

Well, I wasn't aware of that tag. From all I know, installing `pnpm@6` will install the latest v6 anyways.

For `corepack`, I don't know why it says `v16.13` on the `pnpm` site, but the [official docs](https://nodejs.org/api/corepack.html#corepack_corepack) claim it's available from Node.js v16.9. The rest of what you write is true, and yes, this is part of the [MSBuild integration](https://github.com/Lombiq/NodeJs-Extensions/blob/dev/Lombiq.NodeJs.Extensions/build/NodeJsExtensionsNpmPackage.targets#L4).

One great thing I just now saw in the Node.js documentation is the fact that `corepack` is now also available on Node.js **14** beginning with version **14.19**! We should add that to our prerequisites. ([If you can bear how the word looks without the hyphen, use it like that.](https://english.stackexchange.com/a/323664))

Reply:
I've updated this to something sensible.

We can still refine it before merging `task/node-extensions` if necessary.

Reply:
Thanks! I didn't catch that back then.